### ISSUE TITLE
[ARCH LINUX] Excuting error : attempt to index local 'response' ( a nil value )

### ISSUE THREAD

**Issue by isif00** [andweeb/presence.nvim#127]
### Description

I'm using arch linux, a fresh installation, I installed neovim and I've been using NvChad, once I start nvim I get this error, and I had the exact same issue with Zorin os .

### Neovim version

```markdown
NVIM v0.9.1
Build type: Release
LuaJIT 2.1.1692616192

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

### OS information

 Arch Linux x86_64 / gnome 44.4

### Steps to reproduce

I added this: 

```
  {
    'andweeb/presence.nvim',
     ft='presence',
     lazy = false,
  },
  
  ``` 
  under `.config/nvim/lua/custom/plugins.lua` ( following the Nvchad structure )

### Logs

```
Error executing vim.schedule lua callback: ...ocal/share/nvim/lazy/presence.nvim/lua/pr
esence/init.lua:358: attempt to index local 'response' (a nil value)
stack traceback:
        ...ocal/share/nvim/lazy/presence.nvim/lua/presence/init.lua:358: in function 'o
n_done'
        ...l/share/nvim/lazy/presence.nvim/lua/presence/discord.lua:186: in function <.
..l/share/nvim/lazy/presence.nvim/lua/presence/discord.lua:185> ```

**Comment by isif00** [comment]
It worked somehow, not sure why, but i think that i should specify a destination for Neovim to open it, cause opening Neovim without giving the argument path will cause the error .

**Comment by Orlinky** [comment]
Although it does not happen when you open neovim with a target location, I think this is a bug and not an acceptable behavior.

The issue should be re-opened.

**Comment by isif00** [comment]
yes I agree with that, this behavior should not be accepted, but opening Nvim with a target location fix this issue, I'll open it again ...

**Comment by mudejar** [comment]
Any update on this? I am also on Arch Linux, and I believe I'm also triggering the same bug.

**Comment by Yushi5058** [comment]
I am here to look into the same error. I got the same problem on Fedora and actually after a fresh install in Endeavour OS ( Arch Based ). Any update to fix this bug ? Thanks.
